knowledge-base/using-the-html5--viewer-in-mvc-applications-and-binding-data.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Using the HTML5 Viewer in MVC applications and binding data
+description: This article explains how to use the  the HTML5 Viewer in MVC applications and binding data.
```

Comment:
This is not different than the title or the page_title. You must be more concrete and explain what the article is about here. For example, you can add something like: Using data object for reports in the HTML5 Report Viewer within MVC applications.